src/testing/cluster.zig

```diff
@@ -61,6 +62,10 @@ pub fn ClusterType(comptime StateMachineType: anytype) type {
         pub const StateChecker = StateCheckerType(Client, Replica);
         pub const ManifestChecker = ManifestCheckerType(StateMachine.Forest);
 
+        const ClientQueue = RingBuffer(*Message.Request, .{
+            .array = constants.client_request_queue_max,
+        });
```

Comment:
Is this just so that `Cluster.request()`'s API stays the same? It might make more sense to modify the state machine workload so that at most one request per client is ever in flight.